server/tournaments/index.js

```diff
@@ -991,7 +991,6 @@ class Tournament extends Rooms.RoomGame {
 	 */
 	onBattleWin(room, winnerid) {
 		room.tour = null;
-		room.parent = null;
```

Comment:
Actually, why did the old code set this to `null`, anyway? It seems unnecessary.